polaris-react/playground/DetailsPage.module.css

```diff
@@ -1,10 +1,9 @@
-@import '../src/styles/common';
-
 .ContextControl {
   display: flex;
   align-items: center;
   padding: 0 var(--p-space-300);
-  height: $top-bar-height;
+  // stylelint-disable-next-line -- polaris custom global property
+  height: var(--pg-top-bar-height);
```

Comment:
I haven't seen `--pg-` before I couldn't quicky find it in this PR but where are these defined?

Reply:
All the `--pg-` custom properties are [defined in `polaris-react/src/components/AppProvider/global.css`](https://github.com/Shopify/polaris/pull/11677/files#diff-8b82a29970607f740c1b4c6ef27122c4b2218cfd923ea7aaba9db69937c8bc6d) (previously `.scss`) - they're "global" variables (hence the `g`) that used to be global SASS variables and are used across components.

`--pg-` has been added to the linter as a disallowed custom property prefix ([and isn't used anywhere](https://grokt.shopify.io/results?q=%22%5C-%5C-pg%5C-%22) so it's a non-breaking change to the linter).

Reply:
Thanks @jesstelford makes sense to me. I didn't like these as scss variables and I also don't like it now but it makes sense until we refactor some of these global layouts and fix what UI needs these values.